object/application.go

```diff
@@ -544,6 +554,19 @@ func (application *Application) IsPasswordEnabled() bool {
 	}
 }
 
+func (application *Application) IsPasswordWithLdapEnabled() bool {
+	if len(application.SigninMethods) == 0 {
+		return application.EnablePassword
+	} else {
+		for _, signinMethod := range application.SigninMethods {
+			if signinMethod.Name == "Password" && signinMethod.Rule == "All" {
```

Comment:
"None" should also work like "All"

Reply:
Because the None value only affects new users within a week, compatibility with the None option will be dropped